index.html

```diff
@@ -1,188 +1,408 @@
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
     </div>
+  </div>
+ <div class="desktop3">
+  <div class="img">
+    <img class="imgBG" src="./images/desktop 3.png" alt="portfolio card image"></img>
+  </div>
+   <div class="infoCard odd">
+   <section class="info">
+     <h2 class="name">Facebook 360</h2>
+     <ul class="info_job">
+       <li class="job">FACEBOOK</li>
+       <li>Full Stack Dev</li>
+       <li>2015</li>
+     </ul>
+   </section>
+   <section class="decription">
+     <p>
+      Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+     </p>
+   </section>
+   <section class="linkButtons">
+     <button class="link">html</button>
+     <button class="link">Ruby on rails</button>
+     <button class="link">css</button>
+     <button class="link">javaScript</button>
+   </section>
+   <div class="seeMore">
+     <button class="more">See Project</button>
    </div>
+ </div>
+ </div>
+ <div class="desktop4">
+  <div class="img">
+    <img class="imgBG" src="./images/desktop 4.png" alt="portfolio card image"></img>
+  </div>
+   <div class="infoCard">
+   <section class="info">
+     <h2 class="name">Uber Navigation</h2>
+     <ul class="info_job">
+       <li class="job">Uber</li>
+       <li>Lead Developer</li>
+       <li>2018</li>
+     </ul>
+   </section>
+   <section class="decription">
+     <p>
+      A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+     </p>
+   </section>
+   <section class="linkButtons">
+     <button class="link">html</button>
+     <button class="link">Ruby on rails</button>
+     <button class="link">css</button>
+     <button class="link">javaScript</button>
+   </section>
+   <div class="seeMore">
+     <button class="more">See Project</button>
    </div>
-   <div class="container4">
-   <div class="card four">
-    <div class="placeholder-img4"></div>
-    <div class="text">
-        <h3>Multi-Post Stories</h3>
-        <ul class="tech-dev">
-             <li><a href="#">CANOPY</a></li>
-            <li class="tech-dev_dot"></li>
-            <li><a href="#">Back End Dev</a></li>
-            <li class="tech-dev_dot"></li>
-            <li><a href="#">2015</a></li>
+ </div>
+ </section>
+ </div>
+ </section>
+</section>
+    <section class="about-sec" id="about">
+      <div class="content container">
+        <h1 class="heading">About <br> Myself</h1>
+        <p class="body">Hello I'm a software developer! I can help you build a product feature or website. <br>Look
+          through some of
+          my
+          work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
```

Comment:
- [ ] Nicely done with the changes you made in the About section 👌🏽

### How it looks now
![image](https://user-images.githubusercontent.com/43172164/206743709-ebb4bc67-992b-4c29-93bf-4ab692972f90.png)